packages/backend-core/src/redis/utils.ts

```diff
@@ -89,6 +89,7 @@ export function getRedisConnectionDetails() {
 export function getRedisOptions() {
   const { host, password, port } = getRedisConnectionDetails()
   let redisOpts: Redis.RedisOptions = {
+    enableReadyCheck: true,
```

Comment:
This enables the `ready` state in `ioredis` and makes sure we aren't using a Redis connection before it's ready.